01-ruby-fundamentals/07-oo-classes.md

```diff
@@ -17,14 +18,61 @@ Array.new
 # => []
 ```
 
-Where we would normally use the _string literal_ `""`, we can also call the `.new` method on the `String` class to create a new string object. A class can be identified by it's __capitalized first letter__. So `""`, `[]`, and `{}` are syntactic sugar; using _literals_ to _initialize_ a new _instance_ of an _object type_. Most classes do **NOT** have this kind of convenience.
+A class can be identified by it's __capitalized first letter__. In the above snippet `String`, `Hash` and `Array` are all classes. When you call `.new` on a class, you get an _instance_ of that class. In any program, there may be zero, one or many instances of any given class.
+
+While we normally use _string literals_ like `""` or `"elephant"` to create a new string, we can also call the `.new` method on the `String` class to create a new string object. `""`, `[]`, and `{}` are syntactic sugar; using _literals_ to _initialize_ a new _instance_ of an _object type_. Most classes do **NOT** have this kind of convenience.
```

Comment:
Should we even share these examples since (a) we don't encourage them to use these classes this way and (b) they haven't seen them used this way?